zuul.d/edpm.yaml

```diff
@@ -17,40 +28,29 @@
       - openstack-k8s-operators/repo-setup
     roles:
       - zuul: github.com/openstack-k8s-operators/ci-framework
+    pre-run:
+      - ci/playbooks/e2e-prepare.yml
+      - ci/playbooks/dump_zuul_vars.yml
+    post-run:
+      - ci/playbooks/e2e-collect-logs.yml
+      - ci/playbooks/collect-logs.yml
     vars:
-      crc_parameters: "--memory 21000 --disk-size 120 --cpus 8"
-      pre_pull_images:
-        - registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0
+      zuul_log_collection: true
 
 # EDPM job with single node
 - job:
     name: cifmw-crc-podified-edpm-deployment
     parent: cifmw-base-crc-openstack
-    run:
-      - ci/playbooks/e2e-prepare.yml
-      - ci/playbooks/dump_zuul_vars.yml
-      - ci/playbooks/edpm/run.yml
-    post-run:
-      - ci/playbooks/e2e-collect-logs.yml
-      - ci/playbooks/collect-logs.yml
+    run: ci/playbooks/edpm/run.yml
     vars:
       # This should be set here since is associated with parent job (base-crc) and not the scenario
       cifmw_openshift_kubeconfig: "/home/zuul/.crc/machines/crc/kubeconfig"
```

Comment:
From reading comments, it looks like this was needed for the base-crc parent. Do we still need this?

Reply:
Thank you @Sandeepyadav93 for highlighting. It is no longer required here, I have dropped it.

Reply:
thanks for the update.